pkg/splunk/enterprise/configuration.go

```diff
@@ -260,6 +264,9 @@ func getSplunkPorts(instanceType InstanceType) map[string]int {
 	}
 
 	switch instanceType {
+	case SplunkMonitoringConsole:
+		result["hec"] = 8088
```

Comment:
Just a comment, we may need to move some of these hardcoded ports to config later